src/main/asciidoc/_chapters/hbase-default.adoc

```diff
@@ -1949,6 +1949,34 @@ If the DFSClient configuration
 `simple`
 
 
+[[hbase.security.authentication.ui.metrics.protected]]
+*`hbase.security.authentication.ui.metrics.protected`*::
++
+.Description
+
+      Controls whether or not metrics endpoints are allowed only for admin.
+      If true, only users listed on "hbase.security.authentication.spnego.admin.users"
+      or users in group listed on "hbase.security.authentication.spnego.admin.groups"
+      are allowed to access metrics endpoints. (e.g. /jmx, /metrics, /prometheus)
+
++
+.Default
+`false`
+
+
+[[hbase.security.authentication.spnego.kerberos.endpoint.whitelist]]
+*`hbase.security.authentication.spnego.kerberos.endpoint.whitelist`*::
++
+.Description
+
+      Controls whether or not secure authentication is enabled for metrics endpoints.
+      It is valid only when "hbase.security.authentication.ui.metrics.protected" is false.
+
++
+.Default
+`/jmx,/metrics,/prometheus`
```

Comment:
Since we are exposing this as a conf. Don't we need some code change which handles if user overides the conf? What if user wants to do only for /jmx and not others? How is that handled here?

Reply:
I didn't fully understand with your statement.

```
hbase.security.authentication.ui.metrics.protected=false
hbase.security.authentication.spnego.kerberos.endpoint.whitelist=/jmx
```
If an user sets configurations with above, kerberos authentication only for `/jmx` endpoint is disabled.
`/metrics` and `/prometheus` endpoints still needs kerberos authentication. But yes, admin authority doesn't need to access to `/metrics` and `/prometheus` endpoints.
Do you mean I need to add more options to support to control admin authority for each endpoints?

Reply:
I mean to say how are we controlling whitelisting? I do not see any code in this PR using this conf?

Reply:
Got it, we are passing this via params, sorry for the confusion. But I still think it's better to put the value in param explicitly so that at least we ref the whitelisting property and its value. Currently it's happening under the hood and is cryptic and difficult to make out. Let me know WDYT?